LibGAP.jl/src/gap_to_julia.jl

```diff
@@ -0,0 +1,186 @@
+## Converters
+
+## Default
+gap_to_julia(::Type{GAPInputType},x::GAPInputType, recusive_dict = nothing) = x
+gap_to_julia(::Type{Any},         x::GAPInputType, recusive_dict = nothing) = gap_to_julia(x)
+gap_to_julia(::Type{Any},         x::Any         , recusive_dict = nothing) = x
+
+## Integers
+gap_to_julia(::Type{Int128} ,x::Int64, recusive_dict = nothing) = trunc(Int128 ,x)
+gap_to_julia(::Type{Int64}  ,x::Int64, recusive_dict = nothing) = x
+gap_to_julia(::Type{Int32}  ,x::Int64, recusive_dict = nothing) = trunc(Int32  ,x)
+gap_to_julia(::Type{Int16}  ,x::Int64, recusive_dict = nothing) = trunc(Int16  ,x)
+gap_to_julia(::Type{Int8}   ,x::Int64, recusive_dict = nothing) = trunc(Int8   ,x)
+
+## Unsigned Integers
+gap_to_julia(::Type{UInt128},x::Int64, recusive_dict = nothing) = trunc(UInt128,x)
+gap_to_julia(::Type{UInt64} ,x::Int64, recusive_dict = nothing) = trunc(UInt64 ,x)
+gap_to_julia(::Type{UInt32} ,x::Int64, recusive_dict = nothing) = trunc(UInt32 ,x)
+gap_to_julia(::Type{UInt16} ,x::Int64, recusive_dict = nothing) = trunc(UInt16 ,x)
+gap_to_julia(::Type{UInt8}  ,x::Int64, recusive_dict = nothing) = trunc(UInt8  ,x)
+
+## BigInts
+gap_to_julia(::Type{BigInt}, x::Int64, recusive_dict = nothing) = BigInt( x )
+
+function gap_to_julia(::Type{BigInt}, x::MPtr, recusive_dict = nothing)
```

Comment:
According to Codecov, there are no tests for this function at all, it is never called :-(

Reply:
I think the codecov output is wrong. The test is in `conversion.jl`, line 22, and if I remove this function, the tests fail.

Reply:
The problem was that JuliaExperimental tests failed, which caused Travis to stop all further tests, so the LibGAP.jl tests never did run.